pkg/config/config.go

```diff
@@ -102,18 +91,27 @@ type MicroshiftConfig struct {
 	BaseDomain       string        `json:"baseDomain"`
 	Cluster          ClusterConfig `json:"cluster"`
 
-	Ingress IngressConfig      `json:"-"`
-	Etcd    InternalEtcdConfig `json:"etcd"`
+	Ingress IngressConfig `json:"-"`
+	Etcd    EtcdConfig    `json:"etcd"`
 }
 
 // Top level config file
 type Config struct {
-	DNS       DNS        `json:"dns"`
-	Network   Network    `json:"network"`
-	Node      Node       `json:"node"`
-	ApiServer ApiServer  `json:"apiServer"`
-	Debugging Debugging  `json:"debugging"`
-	Etcd      EtcdConfig `json:"etcd"`
+	DNS       DNS       `json:"dns"`
+	Network   Network   `json:"network"`
+	Node      Node      `json:"node"`
+	ApiServer ApiServer `json:"apiServer"`
+	Debugging Debugging `json:"debugging"`
+	Etcd      Etcd      `json:"etcd"`
+}
+
+const (
+	// Etcd performance degrades significantly if the memory available is less than 50MB, enfore this minimum.
+	EtcdMinimumMemoryLimit = "50M"
```

Comment:
In the team meeting today we discussed making this 128M to start. 

```suggestion
	EtcdMinimumMemoryLimit = "128M"
```

Reply:
This variable is the minimum allowed value, not the default. If the user tries to configure etcd to limit it to less than 50MB, Microshift will error.

Reply:
Got it.